src/examples/fetch_live.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from 'expect';
```

Comment:
i think this file could use some spacing and line breaks

Reply:
Will reformat it, thanks.